### PR TITLE
pkg/datasource: Hide enriched timestamp column by default

### PR DESCRIPTION
**Pull request by blanquicet**
# Hide enriched timestamp column by default

Hide the timestamp columns as it takes a lot of space and clutter the screen.

### CODE REVIEW

File: pkg/datasource/columns.go [hunk 1]
```diff
@@ -222,6 +222,8 @@ var annotationsTemplates = map[string]map[string]string{
 		ColumnsWidthAnnotation:    "35",
 		ColumnsMaxWidthAnnotation: "35",
 		ColumnsEllipsisAnnotation: "end",
+		ColumnsHiddenAnnotation:   "true",
```
Comment (burak-ok) [inline pkg/datasource/columns.go:225]: I would prefer to add this annotation in the `formatters.go` where the timestamp field is created

Comment (alban) [reply pkg/datasource/columns.go:225]: Can you please clarify what difference does it make? The purpose of this patch is to hide timestamp.

Comment (burak-ok) [reply pkg/datasource/columns.go:225]: When a gadget developer uses the template in the `gadget.yaml` for his field he wants and gets the correct formatting of the timestamp field itself.

1. `ColumnsHiddenAnnotation:   "true",`
    His field gets hidden by default even if he didn't use the `gadget_timestamp` type

2. Doing it in `formatters.go`
   The field doesn't get hidden if he doesn't use `gadget_timestamp`. If he uses the type, then his field gets hidden.

After a little bit of further thinking I don't know if the use case in 2) makes sense

Comment (mauriciovasquezbernal) [reply pkg/datasource/columns.go:225]: I think it's fine to add it in the template, it'll allow to keep all field configuration on the same place, i.e using "template: timestamp" from the metadata of from the API when adding the field will have the same result. The user can always explicit mark the field as non hidden by using "columns.hidden: false" in the metadata.

**Review by flyth — APPROVED** [review]
LGTM from inspection.

**Pull request merged**